### PR TITLE
flow 68

### PR DESCRIPTION
**Pull request by chrisnojima**
Updates us to flow 68 which finds a ton more issues. 
I also removed a bunch of our own flow lib placeholders that were in libs/flow-interface.js.flow and switched to using flow-typed stubs (do `flow-typed create-stub xxx`)

### CODE REVIEW

File: shared/folders/files/render.desktop.js [hunk 1]
```diff
@@ -209,20 +208,6 @@ class FilesRender extends Component<Props> {
       >
         <Box style={{...globalStyles.flexBoxRow, ...styleHeaderThemed[this.props.theme], height: 48}}>
           <BackButton onClick={this.props.onBack} style={{marginLeft: 16}} />
-          {this.props.recentFilesEnabled && (
```
Comment (chrisnojima) [inline shared/folders/files/render.desktop.js]: was never set

File: shared/reducers/chat2.js [hunk 2]
```diff
@@ -57,11 +57,9 @@ const metaMapReducer = (metaMap, action) => {
       )
     case Chat2Gen.metaRequestingTrusted:
       return metaMap.withMutations(map =>
-        (action.payload.force
```
Comment (chrisnojima) [inline shared/reducers/chat2.js]: force is in another action , not this one

File: shared/tracker/action.desktop.js [hunk 3]
```diff
@@ -3,10 +3,25 @@ import React, {PureComponent} from 'react'
 import {Button, Text, Icon} from '../common-adapters'
 import {globalStyles, globalColors, globalMargins, desktopStyles} from '../styles'
 import {normal} from '../constants/tracker'
+import type {SimpleProofState} from '../constants/types/tracker'
```
Comment (chrisnojima) [inline shared/tracker/action.desktop.js]: update the naming convention to modern and kill all the .native files which were empty renders anyways

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/actions/chat2/index.js [hunk 4]
```diff
@@ -715,7 +715,7 @@ const loadMoreMessages = (
         messageTypes: loadThreadMessageTypes,
       },
       reason:
-        action.type.reason === 'push'
+        reason === 'push'
```
Comment (mmaxim) [inline shared/actions/chat2/index.js]: I noticed in the logs I wasn't seeing this, I guess this is the fix?

Comment (chrisnojima) [reply shared/actions/chat2/index.js]: yeah

**Pull request merged**